nvflare/apis/event_type.py

```diff
@@ -73,8 +73,14 @@ class EventType(object):
 
     BEFORE_CLIENT_REGISTER = "_before_client_register"
     AFTER_CLIENT_REGISTER = "_after_client_register"
-    CLIENT_REGISTERED = "_client_registered"
+    RECEIVED_CLIENT_REGISTER = "_received_client_register"
+    PROCEEDED_CLIENT_REGISTER = "_proceeded_client_register"
```

Comment:
What does this mean: "PROCEEDED_CLIENT_REGISTER"? Do you mean "PROCESSED"?

Reply:
renamed to CLIENT_HEARTBEAT_RECEIVED and CLIENT_HEARTBEAT_PROCESSED